docs/FAQ.md

```diff
@@ -0,0 +1,283 @@
+# FAQ: Frequently Asked Questions about YourBench
+
+Welcome to the **YourBench** FAQ! This document aims to answer common questions about what YourBench is, how it works, and how you can use it to generate dynamic, document-grounded evaluation sets for Large Language Models. Below, you'll find practical information on installation, configuration, usage, and more.
+
+---
+
+## 1. What Is YourBench?
+
+**YourBench** is an open-source framework designed to **generate new, domain-specific or up-to-date benchmarks** for evaluating Large Language Models (LLMs). Instead of relying on static (and often outdated or contaminated) benchmarks, YourBench takes _your_ custom documents—such as company reports, specialized academic texts, or newly published web content—and **automatically creates fresh question-answer pairs** to test models on relevant content. By doing so, it helps you:
+
+- Avoid contamination with older, widely used datasets.
+- Generate relevant evaluations for specialized or emerging topics.
+- Automatically produce large volumes of questions with minimal human labor or cost.
+
+YourBench also supports advanced pipeline stages like summarization, chunking, multi-hop question generation, citation filtering, and more, as detailed below.
+
+---
+
+## 2. What Does the General Pipeline Look Like?
+
+YourBench’s pipeline typically consists of these stages (in a default order):
+
+1. **Ingestion**  
+   Converts raw documents (PDF, MD, HTML, DOCX, etc.) into a normalized Markdown/text format.
+
+2. **Upload/Save**  
+   Optionally packages ingested documents into a Hugging Face Dataset and pushes it to the Hub (or saves locally).
+
+3. **Summarization**  
+   Creates short summaries of each document or chunk to provide global context.
+
+4. **Chunking**  
+   Splits documents into manageable single-hop segments (e.g., by tokens or semantically) and optionally creates multi-hop groupings.
+
+5. **Single-Shot Question Generation**  
+   Generates questions from individual chunk(s) plus global summary.
+
+6. **Multi-Hop Question Generation**  
+   Generates more complex, integrative questions by referencing multiple chunks.
```

Comment:
Needs to explain more what the diff is about - they are the same step technically, but with different processes